src/main/java/com/starbucks/repository/OrderRepository.java

```diff
@@ -1,12 +1,12 @@
 package com.starbucks.repository;
 
 import com.starbucks.model.Order;
-import org.springframework.stereotype.Repository;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
 
 import java.util.UUID;
 
 @Repository
-public interface OrderRepository extends CrudRepository < Order, UUID > {
+public interface OrderRepository extends CrudRepository< Order, UUID > {
```

Comment:
still redundant spaces before Order and after UUID. I'm not sure check in Todo please.

Reply:
Yes